src/components/Popups/FailedNetworkSwitchPopup.tsx

```diff
@@ -12,26 +12,33 @@ const RowNoFlex = styled(AutoRow)`
   flex-wrap: nowrap;
 `
 
+const ColumnContainer = styled(AutoColumn)`
+  margin: 0 12px 0;
+`
+
+export const PopupAlertTriangle = styled(AlertTriangleFilled)`
+  flex-shrink: 0;
+  color: ${({ theme }) => (theme.darkMode ? theme.backgroundSurface : 'white')};
```

Comment:
this is a pretty uncommon situation--should double check w/ design that there's a theme exception here

Reply:
you made me realize i didn't need this line anymore thanks